devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy

```diff
@@ -79,7 +79,12 @@ def on() {
 
 def refresh() {
     Integer reportIntervalMinutes = 5
-    zigbee.onOffRefresh() + zigbee.simpleMeteringPowerRefresh() + zigbee.electricMeasurementPowerRefresh() + zigbee.onOffConfig(0,reportIntervalMinutes * 60) + zigbee.simpleMeteringPowerConfig() + zigbee.electricMeasurementPowerConfig()
+    def cmds = zigbee.onOffRefresh() + zigbee.simpleMeteringPowerRefresh() + zigbee.electricMeasurementPowerRefresh() + zigbee.onOffConfig(0, reportIntervalMinutes * 60) + zigbee.simpleMeteringPowerConfig() + zigbee.electricMeasurementPowerConfig()
+    if (device.getDataValue("manufacturer") == "Jasco Products") {
+        // Some versions of hub firmware will incorrectly remove this binding causing manual control of switch to stop working
+        cmds << ["zdo bind 0x${device.deviceNetworkId} 2 1 0x0006 {${device.zigbeeId}} {${device.zigbeeId}}"]
```

Comment:
We are double binding the on/off cluster on configure, but only doing it once for level.  I don't think we need this in 2 places.